src/runtime_src/core/pcie/tools/xbmgmt/cmd_config.cpp

```diff
@@ -161,23 +161,39 @@ static void showDaemonConf(void)
     writeConf(std::cout, config);
 }
 
-static void showDevConf(std::shared_ptr<pcidev::pci_device>& dev)
+static bool isSupported(std::shared_ptr<pcidev::pci_device>& dev)
 {
-    std::string errmsg, svl;
-    int lvl = 0;
-    bool is_mfg = false;
+    std::string errmsg;
+    bool is_mfg = false, is_recovery = false;
 
     dev->sysfs_get("", "mfg", errmsg, is_mfg, false);
     if (!errmsg.empty()) {
-        std::cerr << "Unexpected error: " << errmsg << std::endl;
-	return;
+        std::cerr << "unexpected error: " << errmsg << std::endl;
+	return false;
     }
 
-    if (is_mfg) {
-        std::cerr << "This operation is not supported with golden image" << std::endl;
-        return;
+    dev->sysfs_get("", "recovery", errmsg, is_recovery, false);
+    if (!errmsg.empty()) {
+        std::cerr << "unexpected error: " << errmsg << std::endl;
+	return false;
+    }
```

Comment:
To support backward compatibility, for that cards which do not have "recovery", should we silently ignore the sysfs node if it is not present?

Reply:
there will always be a sysfs node "recovery" but the value will be set to "0".

Reply:
Oh ok